### PR TITLE
[PR] - Setup cypress testing

### PR DESCRIPTION
**Pull request by erikwesterberg**
-NPM install cypress,
-Adds neccesarly scripts,
-Deleted exemple files in cypress,
-Run cypress

PT - https://www.pivotaltracker.com/n/projects/2384164

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (oliverochman) [inline cypress/fixtures/example.json]: do you need this example file?

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: cypress/fixtures/example.json (hunk 1 above)
Comment (erikwesterberg) [reply cypress/fixtures/example.json]: Nope, its removed and PR updated

File: example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (oliverochman) [inline example.json]: still here :)

Comment (oliverochman) [reply example.json]: please remove this file

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: example.json (hunk 2 above)
Comment (erikwesterberg) [reply example.json]: Removed

File: package.json [hunk 3]
```diff
@@ -8,10 +8,12 @@
     "react-scripts": "3.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "features": "jest -c jest.config.js",
```
Comment (oliverochman) [inline package.json]: why did you add this script?

Comment (erikwesterberg) [reply package.json]: We had it in the cooper challange and thought it was relevent here as well. Its job of what i understan is to make it simple to use mocks outside of the test scope. Maybe thats for later or we don't need it at all, I'll remove it!

File: example.json (hunk 2 above)
Comment (erikwesterberg) [reply example.json]: Removed!
